backend/main.py

```diff
@@ -814,19 +819,12 @@ def filter_pipeline(payload, user):
             print(f"Connection error: {e}")
 
             if r is not None:
-                try:
-                    res = r.json()
-                except:
-                    pass
+                res = r.json()
```

Comment:
If this throws an exception, `res` would be unbound anyway so it would throw on the next one